### PR TITLE
PWN-9197 - Add metadata ext, and add saving ext in database(Part 1)

### PR DESCRIPTION
**Pull request by Davran331122**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-9197

### CODE REVIEW

File: core/src/main/java/org/p2p/core/token/Token.kt [hunk 1]
```diff
@@ -42,14 +42,15 @@ sealed class Token constructor(
         val totalInUsd: BigDecimal?,
         val total: BigDecimal,
         val visibility: TokenVisibility,
+        val extensions: TokenExtension?,
```
Comment (gslevinkov) [inline core/src/main/java/org/p2p/core/token/Token.kt:45]: TokenExtension -> TokenExtensions

File: app/src/main/java/org/p2p/wallet/user/local/TokenListResponse.kt [hunk 2]
```diff
@@ -1,42 +0,0 @@
-package org.p2p.wallet.user.local
-
-import com.google.gson.annotations.SerializedName
-
-data class TokenListResponse(
-
-    @SerializedName("tokens")
-    val tokens: List<TokenResponse>
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/user/local/TokenListResponse.kt:8]: Why this class is deleted? 
It's being used while getting token metadata

File: ethereumkit/src/main/java/org/p2p/ethereumkit/external/repository/EthereumKitRepository.kt [hunk 3]
```diff
@@ -99,11 +98,14 @@ internal class EthereumKitRepository(
                 )
             }
                 .filter { token ->
-                    val tokenBundle = claimingTokens.firstOrNull { token.publicKey == it.contractAddress.hex }
-                    val tokenFiatAmount = token.totalInUsd.orZero()
-                    val isClaimInProgress = tokenBundle != null && tokenBundle.isClaiming
-                    tokenFiatAmount >= MINIMAL_DUST || isClaimInProgress
+                    true
+//                    val tokenBundle = claimingTokens.firstOrNull { token.publicKey == it.contractAddress.hex }
```
Comment (gslevinkov) [inline ethereumkit/src/main/java/org/p2p/ethereumkit/external/repository/EthereumKitRepository.kt]: oops?

File: app/src/main/java/org/p2p/wallet/user/local/TokenListResponse.kt (hunk 2 above)
Comment (Davran331122) [reply app/src/main/java/org/p2p/wallet/user/local/TokenListResponse.kt:8]: yeah, we have copy of this class in token-service module

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**